src/modules/parameters.cpp

```diff
@@ -87,7 +87,7 @@ static PT_THREAD(parametersThreadMain(struct pt* pt))
       PT_WAIT_UNTIL(pt, storageHalRead(PARAMETERS_BANK(!parametersAddress),
                                        &tmpParameters[1],
                                        sizeof(tmpParameters[1])) != HAL_IN_PROGRESS);
-      if (!memcmp(&tmpParameters[0], &tmpParameters[1], sizeof(tmpParameters[0]))) {
+      if (memcmp(&tmpParameters[0], &tmpParameters[1], sizeof(tmpParameters[0]))) {
```

Comment:
This looks good!

[Reference I used](http://www.cplusplus.com/reference/cstring/memcmp/)

Reply:
Even with the newest commit, this seems good.